src/shared/awsagent/agent.go

```diff
@@ -18,20 +26,31 @@ type Agent struct {
 
 func NewAWSAgent(
 	ctx context.Context,
-	oidcUrl string, // TODO: use eks.DescribeCluster to get the OIDC URL.
+	oidcUrlFromEnv string,
```

Comment:
Initials (like OIDC, URL) should use the correct case: https://google.github.io/styleguide/go/decisions#initialisms